rdt/transformers/__init__.py

```diff
@@ -2,61 +2,54 @@
 
 import importlib
 import json
-import os
+import sys
 from collections import defaultdict
 from copy import deepcopy
 from functools import lru_cache
-from glob import glob
+from pathlib import Path
 
 import numpy as np
 
 from rdt.transformers.base import BaseTransformer
 from rdt.transformers.boolean import BooleanTransformer
-from rdt.transformers.categorical import (
-    CategoricalTransformer, LabelEncodingTransformer, OneHotEncodingTransformer)
+from rdt.transformers.categorical import CategoricalTransformer
 from rdt.transformers.datetime import DatetimeTransformer
 from rdt.transformers.null import NullTransformer
-from rdt.transformers.numerical import GaussianCopulaTransformer, NumericalTransformer
+from rdt.transformers.numerical import NumericalTransformer
 
+__all__ = [
+    'BaseTransformer',
+    'NullTransformer',
+    'load_transformer',
+    'get_transformers_by_type',
+    'get_default_transformers',
+    'get_default_transformer',
+]
 
-def _load_addons():
-    addons = []
-    addons_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'addons')
-    for addon_json_path in glob(f'{addons_path}/*/*.json'):
+
+def _import_addons():
+    """Import all the addon modules."""
+    addons_path = Path(__file__).parent / 'addons'
+    for addon_json_path in addons_path.glob('*/*.json'):
         with open(addon_json_path, 'r', encoding='utf-8') as addon_json_file:
             transformers = json.load(addon_json_file).get('transformers', [])
             for transformer in transformers:
                 transformer = transformer.split('.')
                 module = '.'.join(transformer[:-1])
```

Comment:
I know this is my old code but I think we can change it to `rsplit` which is better:
```python3
module = transformer.rsplit('.', 1)[0]
```